edx_exams/apps/core/models.py

```diff
@@ -136,6 +137,14 @@ class Meta:
         db_table = 'exams_examattempt'
         verbose_name = 'exam attempt'
 
+    @classmethod
+    def get_latest_user_exam_attempt(cls, user, exam):
```

Comment:
Question: I'm just getting the latest exam attempt but do I need to consider the status or start_time of the exam attempt to get the most recent, current attempt?

Reply:
latest created sounds about right. As a NIT I think this function might be clearer if we call it 'current' exam attempt since that's the language the UI uses for this.

user_id, and exam_id might be better input parameters here. Calls to this function are more likely to have the id's in scope than whole object.

Reply:
do we want to only get attempts that aren't yet expired?